contracts/SportMarkets/Parlay/ParlayMarketsAMM.sol

```diff
@@ -594,39 +583,6 @@ contract ParlayMarketsAMM is Initializable, ProxyOwned, ProxyPausable, ProxyReen
         sUSD.safeTransfer(safeBox, safeBoxAmount - referrerShare);
     }
 
-    function _buyPositionsFromSportAMM(
-        address[] memory _sportMarkets,
-        uint[] memory _positions,
-        uint[] memory _proportionalAmounts,
-        uint _additionalSlippage,
-        address _parlayMarket,
-        address _parlayOwner
-    ) internal {
-        uint numOfMarkets = _sportMarkets.length;
-        ISportsAMM.Position sportPosition;
-        for (uint i = 0; i < numOfMarkets; i++) {
-            sportPosition = _obtainSportsAMMPosition(_positions[i]);
-            sportsAmm.buyFromAMM(
-                _sportMarkets[i],
-                sportPosition,
-                _proportionalAmounts[i],
-                MAX_APPROVAL,
-                _additionalSlippage
-            );
-            _sendPositionsToMarket(_sportMarkets[i], _positions[i], _parlayMarket, _proportionalAmounts[i]);
-            _updateMarketData(_sportMarkets[i], _positions[i], _parlayMarket, _parlayOwner);
-        }
-    }
-
-    function _updateMarketData(
-        address _market,
-        uint _position,
-        address _parlayMarket,
-        address _parlayOwner
-    ) internal {
-        IParlayMarketData(parlayMarketData).addParlayForGamePosition(_market, _position, _parlayMarket, _parlayOwner);
-    }
-
     function _sendPositionsToMarket(
```

Comment:
this can be removed to save size of ParlayAMM

Reply:
indeed